crates/typst-library/src/compute/data.rs

```diff
@@ -119,11 +123,44 @@ pub fn csv(
     let Spanned { v: path, span } = path;
     let id = vm.location().join(&path).at(span)?;
     let data = vm.world().file(id).at(span)?;
+    csv_decode(Spanned::new(Readable::Bytes(data), span), delimiter)
+}
 
+/// Reads structured data from a CSV string/bytes.
+///
+/// The CSV string/bytes will be read and parsed into a 2-dimensional array of strings:
+/// Each row in the CSV file will be represented as an array of strings, and all
+/// rows will be collected into a single array. Header rows will not be
+/// stripped.
```

Comment:
General comment: I don't think it's necessary to copy paste the whole documentation here. It will be on the same docs page as the main `csv` function where everything is explained.